MIGRATION.md

```diff
@@ -74,8 +75,73 @@
     - [Packages renaming](#packages-renaming)
     - [Deprecated embedded addons](#deprecated-embedded-addons)
 
+
 ## From version 5.2.x to 5.3.x
 
+### To tri-config
+
+In storybook 5.3 3 new files for configuration were introduced, that replaced some previous files.
+
+These files are now soft-deprecated, (*they still work, but over time we will promote users to migrate*):
+
+- `config.js` has been renamed to `preview.js`.
+- `addons.js` has been renamed to `manager.js`.
+- `presets.js` has been renamed to `main.js`.
+
+#### Using main.js
+
+this is what a basic `main.js` looks like:
+
+```js
+module.exports = {
+  stories: ['../**/*.stories.js'],
+  addons: [
+    '@storybook/addon-docs/register',
+  ],
+};
+```
+
+You remove all "register" import from `addons.js` and place them inside the array. If this means `addons.js` is now empty for you, it's safe to remove.
+
+Next you remove the code that imports/requires all your stories from `config.js`, and change it to a glob-pattern and place that glob in the `stories` array. If this means `config.js` is empty, it's safe to remove.
+
+If you had a `presets.js` file before you can add the array of presets to the main.js file and remove `presets.js` like so:
+
+```js
+module.exports = {
+  stories: ['../**/*.stories.js'],
+  presets: ['@storybook/addon-docs/preset'],
+};
+```
+
+#### Using preview.js
+
+If after migrating the imports/requires of your stories to `main.js` you're left with some code in `config.js` it's likely the usage of `useParameters` & `addDecorator`.
+
+This is fine, rename `config.js` to `preview.js`.
+
+This file can also be used to inject global stylesheets, fonts etc, into the preview bundle.
+
+#### Using manager.js
+
+If you are setting storybook options in `config.js`, especially `theme`, you should migrate it to `manager.js`:
```

Comment:
Add a sentence on why this is an improvement

Reply:
✅